src/dsp_tools/utils/project_validate.py

```diff
@@ -13,7 +13,56 @@
 from dsp_tools.utils.excel_to_json_lists import expand_lists_from_excel
 
 
-def check_for_undefined_super_resource(project_definition: dict[str, Any]) -> bool:
+def _check_for_duplicate_names(project_definition: dict[str, Any]) -> bool:
```

Comment:
there seems to be a lot of similar stuff going on between `excel_to_json_properties.py`, `excel_to_json_resources.py` and here. Could this be unified somehow?

Reply:
also, the method seems to only use `project_definition["project"]["ontologies"]`, could you pass in only that? As a rule of thumb, it's always best to give a function only access to the things it needs, so that only changes to that can break the function.